src/rpcblockchain.cpp

```diff
@@ -1252,6 +1254,10 @@ bool AdvertiseBeacon(bool bFromService, std::string &sOutPrivKey, std::string &s
                 // Store the key 
                 sMessage = AddContract(sType,sName,sBase);
                 StoreBeaconKeys(GlobalCPUMiningCPID.cpid, sOutPubKey, sOutPrivKey);
+                // Backup config with new keys with beacon suffix
+                std::string sBeaconBackupConfigFilename = GetBackupFilename("gridcoinresearch.conf", "beacon");
+                boost::filesystem::path sBeaconBackupConfigTarget = GetDataDir() / "walletbackups" / sBeaconBackupConfigFilename;
+                BackupConfigFile(sBeaconBackupConfigTarget.string().c_str());
```

Comment:
oki ill make the change but then it shouldnt just be suffic -beacon. seeing with ur change before u can't overwrite keys anymore unless beacon is successful? thus the try {